docs/getting-started/local-setup/creating-a-keypair.md

```diff
@@ -0,0 +1,75 @@
+---
+id: creating-a-keypair
+---
+
+# Creating a Keypair
```

Comment:
@aeddi, since you're new in the team, please, review this tutorial and PR, thank you.